spec/factories/task.rb

```diff
@@ -459,6 +459,25 @@ def self.find_first_task_or_create(appeal, task_type, **kwargs)
         assigned_to { VhaBusinessLine.singleton }
       end
 
+      factory :higher_level_review_vha_task_on_hold, class: DecisionReviewTask do
+        appeal do
+          FactoryBot.create(:higher_level_review,
```

Comment:
You will have to fix the alignment issues that are caused by this suggestion if you accept it.
```suggestion
          create(:higher_level_review,
```